livedata/src/test/java/io/getstream/chat/android/livedata/repository/QueryChannelsImplRepositoryTest.kt

```diff
@@ -17,7 +17,7 @@ class QueryChannelsImplRepositoryTest : BaseDomainTest() {
     @Test
     fun testInsertAndRead() = runBlocking(Dispatchers.IO) {
         val queryChannelsEntity = QueryChannelsEntity(data.filter1, QuerySort())
-        queryChannelsEntity.channelCids = sortedSetOf("a", "b", "c")
+        queryChannelsEntity.channelCids = listOf("a", "b", "c")
```

Comment:
be sure to test in the sample app that we don't get the same channel showing up multiple times...

Reply:
It is handled when new channels are added